INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': 1ebce099-1834-4874-a2c5-b60318297c0f, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': 1ebce099-1834-4874-a2c5-b60318297c0f, 'url': None, 'current_dataset_version_id': aec438f0-1abb-4cfe-83b2-6203c1b088e8}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [1049a771-0bdf-4399-97a5-ad40a85b49a4/raw.h5ad] to s3://artifact_bucket/1049a771-0bdf-4399-97a5-ad40a85b49a4/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
ERROR    processing:dataset_metadata_update.py:261 Cannot find labeled H5AD artifact uri for aec438f0-1abb-4cfe-83b2-6203c1b088e8.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:265 Main: Starting thread for rds update
INFO     processing:dataset_metadata_update.py:288 Main: Starting thread for cxg update
INFO     processing:process_logic.py:45 Updating processing status